webapp-django/crashstats/crashstats/models.py

```diff
@@ -761,7 +761,6 @@ class RawCrash(SocorroMiddleware):
     API_ALLOWLIST = (
         # Crash annotations:
         # https://searchfox.org/mozilla-central/source/toolkit/crashreporter/CrashAnnotations.yaml
-        "additional_minidumps",
```

Comment:
This field was in the list twice. Further, it's not even something that's in the raw crash, so I'm not sure why it was in there at all.